environments/prod/main.tf

```diff
@@ -84,11 +95,32 @@ module "container_orchestrator" {
   }
 
   network = {
-    # checkov:skip=CKV_AWS_88: "EC2 instance should not have public IP."
-    # For cost saving purpose we place instances in public subnet and assign public IP.
-    # Disabling public ips would require NAT or PrivateLink to communicate with ECS
-    assign_instance_public_ips = true
-    subnets                    = module.network.public_subnets
+    assign_instance_public_ips = false
+    subnets                    = module.network.private_subnets
```

Comment:
Check if the EC2 instance is attached to ECS after that.

Reply:
yes is working well - already deployed